wgsl/index.bs

```diff
@@ -1305,6 +1305,16 @@ Note: When no conversion is performed, the conversion rank is zero.
       <td>f16
       <td>7
       <td>Behaves as [=AbstractInt=] to [=AbstractFloat=], and then [=AbstractFloat=] to f16
+  <tr algorithm="conversion rank abstract vector">
```

Comment:
Should this remove the subsequent paragraph about vectors and matrices?

Reply:
Done.